apps/web/src/utils/badge.tsx

```diff
@@ -0,0 +1,69 @@
+import Tooltip from "@web/components/Tooltip";
+import { Permer } from "permer";
+import { Code, Plus, Tool, Zap } from "react-feather";
+import { SelfUser } from "../../../../packages/commons/types";
+import { styled } from "@web/stitches.config";
+
+const badges = [
+  {
+    id: "admin",
+    tooltip: "imperialb.in Staff",
+    icon: <Tool color="dark red" fill="red" />,
+  },
+
+  {
+    id: "beta-tester",
+    tooltip: "Beta Tester",
+    icon: <Code color="dark purple" fill="purple" />,
+  },
+
+  {
+    id: "early-adopter",
+    tooltip: "Early Adopter",
+    icon: <Zap color="dark orange" fill="orange" />,
+  },
+
+  {
+    id: "member-plus",
+    tooltip: "Supporter of imperialb.in",
+    icon: <Plus color="pink" fill="pink" />,
+  },
+] as const;
+
+const badgeChecker = new Permer(
+  badges.filter((b) => b.id !== "early-adopter").map((b) => b.id),
+);
+
+const getBadges = (user: SelfUser) => {
+  const b = badges.filter((b) => badgeChecker.test(user.flags, b.id));
+
+  // hacky way to implement early adopter, will always
```

Comment:
```suggestion
  // Hacky way to implement early adopter, will always
```

eslint wont be 2 happy